conf/defaults.ini

```diff
@@ -786,6 +786,9 @@ global_session = -1
 # global limit of alerts
 global_alert_rule = -1
 
+# global limit of files uploaded to the SQL DB
+global_files_in_sql = 1000
```

Comment:
I think it's better to follow existing naming convention without trying to fit future model with pluggable storages. Because seems we will anyway deal with all existing options here – like `global_dashboard`. So we won't be able to migrate without revisiting quotas anyway.

Also, we could have 2 SQL storages theoretically in the future which also sth that ruins understanding of the behavior.

Does it make sense?

Reply:
👍 renamed to `file`